components/style/bloom.rs

```diff
@@ -121,6 +121,7 @@ impl<E: TElement> StyleBloom<E> {
     /// Create an empty `StyleBloom`. Because StyleBloom acquires the thread-
     /// local filter buffer, creating multiple live StyleBloom instances at
     /// the same time on the same thread will panic.
+    #[inline(never)]
```

Comment:
Maybe leave a comment on the reasoning for this here and below?

Reply:
Done.